css/css-flexbox/flexbox_mix-writing-flex-alignment.html

```diff
@@ -0,0 +1,122 @@
+
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script>
+    // Generating script.
+    // const writingModes = [
+    //   'horizontal-tb',
+    //   'vertical-rl',
+    //   'vertical-lr',
+    //   'sideways-rl',
+    //   'sideways-lr'
+    // ];
+    // const alignmentProps = [
```

Comment:
Do we want to split this out even further (into `justify-content`, `align-items`, and `flex-direction`), and test even more potential combinations?

That said, if we're going to have to hard code the expected positions, that probably gets to unreasonably many tests (given it means we'd go from 5*13 to 5*5*5*6)… @gregwhitworth, thoughts?

Reply:
reiterating what @gsnedders is saying, maybe it makes more sense to break these into separate tests. Or, just simplify the tests to ones that _might_ actually break. I probably don't know enough to make that determination.